compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -247,43 +247,13 @@ class ComposeScene internal constructor(
         }
     }
 
-    private val childScenes = mutableSetOf<ComposeScene>()
-
-    /**
-     * Adds a child [ComposeScene], so that nodes in it can be found in tests.
-     */
-    internal fun addChildScene(scene: ComposeScene) {
-        check(!isClosed) { "ComposeScene is closed" }
-        childScenes.add(scene)
-    }
-
-    /**
-     * Removes a child [ComposeScene].
-     */
-    internal fun removeChildScene(scene: ComposeScene) {
-        check(!isClosed) { "ComposeScene is closed" }
-        childScenes.remove(scene)
-    }
-
-    /**
-     * Adds this scene's [RootForTest]s, including any of its child scenes, into the given set.
-     */
-    private fun addRootsForTestTo(target: MutableSet<RootForTest>) {
-        target.addAll(owners)
-        for (child in childScenes) {
-            child.addRootsForTestTo(target)
-        }
-    }
-
-    /**
-     * All currently registered [RootForTest]s. After calling [setContent] the first root
-     * will be added. If there is an any [Popup] is present in the content, it will be added as
-     * another [RootForTest]
-     */
+    @Deprecated(
```

Comment:
There was [a case](https://github.com/JetBrains/compose-multiplatform/issues/753#issuecomment-863021459), when it is useful - to access the tree of semantic nodes (it was also asked by another person in the private conversation - how to make an analog of the layout inspector).

Can we introduce `ComposeScene.semanticOwner` (or `ComposeScene.semanticOwners`, not sure about the end design yet)

Reply:
Hm, agree. And in this case, we can use it in our own accessibility code instead of getting `AccessibilityController` from it inside the owner. But let's make it separately

Reply:
I would introduce it right now, before the merge (we can do it in a separate PR). As there can be questions from users of dev builds, and we can don't have time/forget to do it.

Reply:
Adding it as an Experimental is compromise, as this API has rare uses cases (but still has)